### PR TITLE
deprecate single element relationships

### PR DESCRIPTION
**Pull request by angrykoala**
# Description

Deprecates single item relationships

### CODE REVIEW

File: .changeset/chilled-zebras-help.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+"@neo4j/graphql": patch
+---
+
+Deprecate single element relationships:
```
Comment (darrellwarde) [inline .changeset/chilled-zebras-help.md:5]: I think this changeset is by far not long enough - this needs a lot of explanation as to the rationale IMO!

**Review by darrellwarde — DISMISSED** [review]
I think looks good, but consider my comment?

File: .changeset/chilled-zebras-help.md (hunk 1 above)
Comment (angrykoala) [reply .changeset/chilled-zebras-help.md:5]: Isn't that explanation better left for the migration guide?

Comment (darrellwarde) [reply .changeset/chilled-zebras-help.md:5]: I dunno - I know this isn't the breaking change itself but this is what changesets outputs:

```
🦋  warn This Changeset includes a major change and we STRONGLY recommend adding more information to the changeset:
🦋  warn WHAT the breaking change is
🦋  warn WHY the change was made
🦋  warn HOW a consumer should update their code
```

**Pull request merged**